### DISCUSSION TITLE
How to compile the Fast-dds for QNX platform?

### DISCUSSION THREAD

**Discussion by safrans** [eProsima/Fast-DDS#2391] [Unofficial support]
I want to compile Fast-dds for QNX platform, but there are many errors in compilation.
Are there compile for QNX platform relevant document?

**Comment by MiguelCompany** [comment]
QNX has official documentation on how to build ROS 2 for QNX [here](https://ros2-qnx-documentation.readthedocs.io/en/latest/). The build process includes building Fast DDS.

**Comment by simoncj** [comment]
I try to find the build process of building Fast DDS, mentioned in "how to build ROS2 for QNX" by Miguel, but nothing found.  Does any one can help me to show the steps of building Fast DDS for QNX?
